core/controllers/reader.py

```diff
@@ -294,6 +298,96 @@ def get(self, exploration_id):
         self.render_json(self.values)
 
 
+class StorePlaythroughHandler(base.BaseHandler):
+    """Handles a useful playthrough coming in from the frontend to store it."""
+
+    REQUIRE_PAYLOAD_CSRF_CHECK = False
```

Comment:
Please add a comment to explain what this is/what it is used for.

Reply:
Actually @pranavsid98 this seems incorrect. We should have a CSRF check. Why bypass it?

Reply:
Done.